### ISSUE TITLE
isColor does not detect capitalized color keywords

### ISSUE THREAD

**Issue by bartveneman** [shellscape/postcss-values-parser#67]
- Webpack Version: N/A
- Operating System (or Browser): Mac OS 10.14.3
- Node Version: 10.15.2
- postcss-values-parser Version: 3.0.0-beta.4

### How Do We Reproduce?

```js
const {parse} = require('postcss-values-parser')

;['Red', 'GREEN', 'whiteSmoke'].forEach(color => {
  parse(color, {loose: true}).walk(node => {
    console.log(node.isColor)
  })
})
```

### Expected Behavior

Expected `isColor` to be `true` for all of these cases

### Actual Behavior

`isColor` is `false` for all of these cases

### Related issues and code
* Related to #64 
* Relevant usage in @projectwallace: https://github.com/projectwallace/css-analyzer/commit/15a7048f40fbe1d887bac96ac924882b91211ae8#diff-8a256177cb7a89d5370d4e5e404a3bbaR10

**Comment by shellscape** [comment]
Good catch. Should be an easy fix.

**Comment by shellscape** [comment]
Fixed and released in v3.0.0 proper. We're out of beta. Thank you for your help in adopting the beta!

**Comment by bartveneman** [comment]
Whoo! 🎉
